Jenkinsfile

```diff
@@ -81,7 +81,7 @@ pipeline {
                 echo 'Running tests'
                 // all tests is very very long (10 hours on Apache Jenkins)
                 // sh 'mvn -B -e test -pl activemq-unit-tests -Dactivemq.tests=all'
-                sh 'mvn -B -e test'
+                sh 'mvn -fae -B -e test'
```

Comment:
Nice.

Since you're touching these, any opposition to using the long arguments, so those not super familiar with maven can understand what they mean? :)

Reply:
The commit comments cover it ;-)  The short form is appropriate for developer CI/CD tools.